### PR TITLE
add support for Centos 8

### PR DESCRIPTION
**Pull request by fujexo**
##### ISSUE TYPE

 - Feature Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by malugiso — APPROVED** [review]

File: vars/CentOS_8.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+
+# by ansible used packages
```
Comment (tongpu) [inline vars/CentOS_8.yml:3]:
```suggestion
# packages used by ansible
```

**Comment by malugiso** [comment]
tested, works

File: vars/CentOS_8.yml (hunk 1 above)
Comment (tongpu) [reply vars/CentOS_8.yml:3]: Maybe we should also fix the comment `RedHat.yml`.

The same configuration would probably also be relevant for RHEL 8 and we should create a file for that too.

**Review by tongpu — APPROVED** [review]
LGTM

**Review by tongpu — APPROVED** [review]
LGTM

**Pull request merged**